webpay/pay/tasks.py

```diff
@@ -275,13 +280,67 @@ def start_pay(transaction_uuid, notes, user_uuid, provider_names, **kw):
             'status': constants.STATUS_PENDING
         })
     except Exception, exc:
+        etype, val, tb = sys.exc_info()
+        pay_error_handler(
```

Comment:
This should be wrapped in a try/except so that when it fails, we still raise the original exception rather than crashing on trying to report the error.